trl/experimental/gold/gold.py

```diff
@@ -21,34 +21,17 @@
 # ///
 
 """
-# Full training:
-python trl/experimental/gold/gold.py \
-    --model_name_or_path meta-llama/Llama-3.2-1B-Instruct \
-    --teacher_model_name_or_path Qwen/Qwen2-1.5B-Instruct \
-    --dataset_name trl-lib/chatbot_arena_completions \
-    --learning_rate 2e-5 \
-    --per_device_train_batch_size 4 \
-    --gradient_accumulation_steps 8 \
-    --output_dir gold-model \
-    --num_train_epochs 1 \
-    --push_to_hub \
+# Full training: python trl/experimental/gold/gold.py \
```

Comment:
I don't know why it tries to format this, it doesn't do it for the other scripts.
Anyway, you should revert this change